[DOCS] Adding metadata to articles (#18153)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/Converting_Model.md

@sphinxdirective

.. meta:: 

>    :description: When provided an additional shape definition with "input_shape" 

Reviewer: if we want to make call-to-action descriptions, this could also be something like learn how to increase the efficiency of a model with MO by providing an additional shape definition with the "input_shape" and "static_shape" parameters.

> .. meta:: 
>    :description: When provided an additional shape definition with "input_shape" 
>                  and "static_shape" parameters, MO can increase efficiency of a model.

Author: ```suggestion
.. meta:: 
   :description: Learn how to increase the efficiency of a model with MO by providing an additional shape definition with the input_shape and static_shape parameters.
```
